tests/unit_tests/forward_model_runner/test_jobmanager.py

```diff
@@ -123,6 +123,62 @@ def test_run_multiple_ok():
         assert os.path.getsize(f"mkdir_err.{dir_number}") == 0
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+def test_run_multiple_jobs_just_one_manifest():
```

Comment:
Not sure what is being tested here, so some better name of the test would be great.

Reply:
Renamed the tests